server/jetstream_cluster.go

```diff
@@ -1499,7 +1503,9 @@ func (js *jetStream) createRaftGroup(accName string, rg *raftGroup, storage Stor
 		s.Debugf("Error creating raft group: %v", err)
 		return err
 	}
+	js.mu.Lock()
 	rg.node = n
+	js.mu.Unlock()
 
 	// See if we are preferred and should start campaign immediately.
 	if n.ID() == rg.Preferred && n.Term() == 0 {
```

Comment:
We reference rg.Preferred here so I think we still need the lock or need to capture.

Reply:
Ok, I checked n.Campain() and not basically reseting some timer, so I could have "r.node = n" and everything else under the js.mu.Lock(), is that ok?

Reply:
Yes I think so.

Reply:
Ok, done